NitroxModel/Packets/PlayerDeathEvent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using NitroxModel.DataStructures;
+using System;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class PlayerDeathEvent : AuthenticatedPacket
+    {
+        public Vector3 DeathPosition { get; private set; }
```

Comment:
minor: we removed all of the 'private set' as these are not required for the immutable objects.